src/modal/modal.service.ts

```diff
@@ -82,7 +86,7 @@ export class ModalService {
 	 * @returns {ComponentRef<any>}
 	 * @memberof ModalService
 	 */
-	show(data: {modalType?: string, modalLabel?: string, modalTitle: string, modalContent: string, buttons?: null}) {
+	show(data: {modalType?: string, modalLabel?: string, modalTitle?: string, modalContent?: string, buttons?: []}) {
```

Comment:
nice!

if we wanted to get really fancy we could declare a `interface ModalButton` and type `buttons` to `buttons?: ModalButton[]`

Reply:
yeap, would definitely add that in the future!
I'm still an Angular NOOB but also really pressed for our project's release in 3 weeks

Reply:
Cool, I'll make an issue to note that 👍